.circleci/config.yml

```diff
@@ -353,6 +354,20 @@ jobs:
       - run: cd solidity/ && curl "https://binaries.soliditylang.org/bin/soljson-nightly.js" --location --output soljson.js
       - run: cd solidity/ && test/externalTests/solc-js/solc-js.sh "$(realpath soljson.js)" "$(scripts/get_version.sh)" "$(realpath ../solc-js/)"
 
+  download-release-solc-test:
+    docker:
+      - image: cimg/node:current
+    steps:
+      - show-npm-version
+      - provision-and-package-solcjs
```

Comment:
Do we need this step? You seem to be using the repo directly, rather than the package this installs.

Reply:
No, it is not. I used it because of the provision part. I thought about installing it as a dependency but changed my mind and did not update the script.